pulp_2_tests/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -223,6 +223,11 @@ class ChangeFeedTestCase(BaseAPITestCase):
     <https://pulp.plan.io/issues/1922>`_.
     """
 
+    @classmethod
+    def setUpClass(cls):
+        """Set config used by each test case."""
+        cls.cfg = config.get_config()
+
```

Comment:
Since a `setUpClass` was introduced. I suggest take a look in other objects/variables that can be moved here.